go_to_location:
  # - {name: "coke"         , frame: "world", position: [-0.005, 1.500, 1.398], quaternion: [-0.431, 0.560, 0.443, 0.552]} #only as example
  # - {name: "approach_coke", frame: "world", position: [-0.005, 1.500, 1.598], quaternion: [-0.431, 0.560, 0.443, 0.552]} #only as example
  # - {name: "coke_hole_1"  , frame: "world", position: [0.349, 1.008, 1.388] , quaternion: [-0.527, 0.467, 0.540, 0.462]} #only as example


go_to_location_server:

  ik_sol_number: 4
  max_stall_iter: 100
  collision_detector: FCL

  groups:
    manipulation: open_tip

  manipulation:
    preferred_configuration: [0.0, 0.0, 0.0, 0.0, 0.0, 0.0]
    #preferred_configuration: [-1.9769910017596644, -0.8349288266948243, -2.292120933532715, -1.6209961376585902, -1.5518959204303187, 0.5749483108520508]
    preferred_configuration_weight: [1, 1, 1, 1, 1, 1]
    max_ik_goal_number: 32
    use_single_goal: false
    upper_bound: [ 6.29,  6.29, 6.29,  6.29,  6.29,  12.5]
    lower_bound: [-6.29, -6.29,-6.29, -6.29, -6.29, -12.5]


  request_adapters:
  - default_planner_request_adapters/AddTimeParameterization
  - default_planner_request_adapters/FixWorkspaceBounds
  - default_planner_request_adapters/FixStartStateBounds
  - default_planner_request_adapters/FixStartStateCollision
  - default_planner_request_adapters/FixStartStatePathConstraints
